apps/passport-client/components/screens/LoginScreens/LoginScreen.tsx

```diff
@@ -154,6 +165,12 @@ export function LoginScreen(): JSX.Element {
           storage: storageRequest.value,
           encryptionKey
         });
+      } else {
```

Comment:
I think it would be cleaner/clearer if this function returned a boolean and the else clause happened in the caller, but I think the current logic is correct.